### PR TITLE
[Keyboard] RS60 Rev2 Addition

### PR DESCRIPTION
**Pull request by Xelus22**
## Description

Add RS60 Rev2

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Add RS60 Rev2

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/xelus/rs60/rev2/chconf.h [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/*
+ * This file was auto-generated by:
+ *    `qmk chibios-confmigrate -i keyboards/acheron/austin/chconf.h -r platforms/chibios/common/configs/chconf.h`
+ */
+
+#pragma once
```
Comment (drashna) [inline keyboards/xelus/rs60/rev2/chconf.h:22]: Is this file needed, or is it just copied and pasted?

**Review by drashna — APPROVED** [review]

File: keyboards/xelus/rs60/rev2/chconf.h (hunk 1 above)
Comment (Xelus22) [reply keyboards/xelus/rs60/rev2/chconf.h:22]: I've copy pasted, since i've looked at the Mode/m75s files which uses this. Unsure if this is required.

**Review by fauxpark — APPROVED** [review]

File: keyboards/xelus/rs60/rev2/chconf.h (hunk 1 above)
Comment (drashna) [reply keyboards/xelus/rs60/rev2/chconf.h:22]: gonna merge for now.

Though, the reason I ask, is that the blackpill boards also seem to have this config file, but don't need it. In fact, it was a bit more stable/faster after removing it.
